pages/en/lb4/Using-components.md

```diff
@@ -7,24 +7,38 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Using-components.html
 summary:
 ---
-Components are an important part of LoopBack 4. We are keeping the core small and easy to extend, making it easy for independent developers to contribute additional features to LoopBack. Components are the vehicle for bringing more goodness to your applications.
 
-A typical LoopBack component is an [npm](https://www.npmjs.com/) package exporting a Component class which can be added to your application.
+One of the many ways LoopBack 4 allows for extensibility is through Components.
+A Component makes it easy for independent developer to contribute additional
+features to LoopBack. Components provide a package that allows you to extends
+your Application.
 
-```js
-import {Application} from '@loopback/core';
-import {AuthenticationComponent} from '@loopback/authentication';
+A typical LoopBack component is an [npm](https://www.npmjs.com) package
+exporting a Component class whih can be added to your application.
 
-const app = new Application();
+```ts
+import { RestApplication } from "@loopback/rest";
+import { AuthenticationComponent } from "@loopback/authentication";
+
+const app = new RestApplication();
+// Add component to Application, which provides bindings for
+// authenticate sequence action
 app.component(AuthenticationComponent);
 ```
 
-In general, components can contribute the following items:
+Components can contribute the following items:
+
+* [Controllers](Controllers.html)
+* Providers of additional [Context values](Context.html)
 
- - [Controllers](Controllers.html)
- - Providers of additional [Context values](Context.html)
+Certain Components may require the use of a (Mixin)[Mixin.html] so they may
```

Comment:
Brackets are switched up: `[Mixin](Mixin.html)`

Reply:
This section isn't clear enough for my liking. How is the mixin used along with the component? Do I still need to register my component if I'm using the mixin?

Reply:
I tried re-phrasing it, PTAL and let me know if it makes any more sense or not.